doc/source/api/solver/fieldinfo.rst

```diff
@@ -0,0 +1,98 @@
+.. _ref_field_info:
+
+Field info
+==========
+
+You can use field info objects to access Fluent field information.
+
+Accessing field info objects
+----------------------------
+
+In order to access field info, launch the fluent solver, and make field
+data available (for example, by reading case and data files):
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case-dats", file_name=mixing_elbow_case_path)
+  >>> solver.solution.initialization.hybrid_initialize()
+
+
+The field info object is an attribute of the solver object:
+
+.. code-block:: python
+
+  >>> field_info = solver.field_info
+
+Sample requests
+---------------
+
+Here are the methods for requesting each type of field:
+
+- ``get_range`` for getting the range of the field.
+- ``get_fields_info`` for getting fields information.
+- ``get_vector_fields_info`` for getting vector fields information.
+- ``get_surfaces_info`` for getting the surfaces information.
+
+Get fields info
+~~~~~~~~~~~~~~~
+You can request the fields information (field name, domain, and section) by
+calling the ``get_fields_info`` method.
+
+.. code-block:: python
+
+  >>> field_info.get_fields_info()
+  {'pressure': {'display_name': 'Static Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
+   'pressure-coefficient': {'display_name': 'Pressure Coefficient', 'section': 'Pressure...', 'domain': 'mixture'},
+   'dynamic-pressure': {'display_name': 'Dynamic Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
+   'absolute-pressure': {'display_name': 'Absolute Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
+    ...}
+
+Get range
+~~~~~~~~~
+You can request the range (minimum and maximum values) for a given ``field`` by
+calling the ``get_range`` method. It takes an argument field (along with other
+optional arguments) which can be referred from ``get_fields_info``.
```

Comment:
@prmukherj I'm not sure what you mean by "which can be referred from".

Reply:
Thank you @seanpearsonuk, this has been resolved.